src/main/kotlin/lotto/model/Lottos.kt

```diff
@@ -0,0 +1,33 @@
+package lotto.model
+
+class Lottos(val lottos: List<Lotto>) {
+    constructor(count: Int) : this(createLottos(count))
+
+    fun check(winningNumbers: List<Int>, checkCount: Int): Int {
+        return lottos
+            .map { it.getWinningCount(winningNumbers) }
+            .filter { it == checkCount }
+            .count()
+    }
+
+    fun getEarningRate(winningNumbers: List<Int>): Double {
+        val totalPrizeMoney = getTotalPrizeMoney(winningNumbers)
+        val budgetMoney = lottos.size.times(COST_PER_ONE_LOTTO)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val budgetMoney = lottos.size * COST_PER_ONE_LOTTO
```